apiStar_folder/templates/index1.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+    <body>
+
+        <h2>Textarea for user's input</h2>
+        <p>Type input in env.spec format.</p>
+        <textarea id="myTextarea"></textarea>
+
+        <button type="button" onclick="myFunction()">Try it</button>
+
+        <p id="demo"></p>
+
+        <script>
+        function myFunction() {
```

Comment:
Rename this to something that represents better the functionality of this function please. What about `requestRenderedEnvSpec`?